frontend/src/app/services/subject-comments.service.ts

```diff
@@ -102,21 +103,21 @@ export class SubjectCommentsService {
      * @param {string} errorMessageRequest
      * @returns {Promise<SubjectComment>}
      */
-    commentActionPost(url: string, requestBody: object, errorMessageApi: string, errorMessageRequest: string): Promise<SubjectComment>{
+    commentActionPost(url: string, requestBody: object, errorMessageApi: string, errorMessageRequest: string): Promise<SubjectHubApiResponse>{
         return this.http
-            .post(url,requestBody)
+            .post(environment.API_URL + url, requestBody)
             .toPromise()
             .then(response => {
                 if(response.status == 200){
-                    return response.json() as SubjectComment;
+                    return new SubjectHubApiResponse(true, response.json() as SubjectComment);
```

Comment:
rather than passing a boollean true or false into the SubjectHubApiResponse Constructor you can turn your if statement into the boolean.

i.e. const wasSuccessfull = response.status == 200;

then return new SubjectHubApiResponse(wasSuccessful, response.json())

Reply:
ic, smart movvees. gives it more meaning imo

:dancer: :dango: :dancing_men: :dancing_women: :dango: :man_dancing: